### PR TITLE
Add new style for chair of Prof. Weber

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: universitaet-mannheim-lehrstuhl-prof-weber.csl [hunk 1]
```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Universität Mannheim - Lehrstuhl Prof. Weber</title>
+    <id>http://www.zotero.org/styles/universität-mannheim-lehrstuhl-prof-weber</id>
+    <link href="http://www.zotero.org/styles/universität-mannheim-lehrstuhl-prof-weber" rel="self"/>
```
Comment (zuphilip) [inline universitaet-mannheim-lehrstuhl-prof-weber.csl]: Umlaut `ä` must be encoded as `a` and also the filename has then to match here.

File: universitaet-mannheim-lehrstuhl-prof-weber.csl [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    <link href="http://www.zotero.org/styles/universitat-mannheim-lehrstuhl-prof-helmig" rel="template"/>
+    <link href="http://weber.bwl.uni-mannheim.de/fileadmin/files/weber/files/secret/Masterarbeiten/Wissenschaft_Arbeiten_September_2015_deutsch_neu.pdf" rel="documentation"/>
+    <link href="http://weber.bwl.uni-mannheim.de/fileadmin/files/weber/files/secret/Masterarbeiten/Wissenschaft_Arbeiten_September_2015_english_neu-1.pdf" rel="documentation"/>
+    <link href="http://weber.bwl.uni-mannheim.de/fileadmin/files/weber/files/teaching/Wissenschaft_Arbeiten_Mai_2010.pdf" rel="documentation"/>
+    <author>
+      <name>Philipp Zumstein</name>
+      <uri>https://github.com/zuphilip</uri>
+    </author>
+    <category citation-format="note"/>
```
Comment (zuphilip) [inline universitaet-mannheim-lehrstuhl-prof-weber.csl]: Nope

File: universitaet-mannheim-lehrstuhl-prof-weber.csl [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    <link href="http://weber.bwl.uni-mannheim.de/fileadmin/files/weber/files/secret/Masterarbeiten/Wissenschaft_Arbeiten_September_2015_english_neu-1.pdf" rel="documentation"/>
+    <link href="http://weber.bwl.uni-mannheim.de/fileadmin/files/weber/files/teaching/Wissenschaft_Arbeiten_Mai_2010.pdf" rel="documentation"/>
+    <author>
+      <name>Philipp Zumstein</name>
+      <uri>https://github.com/zuphilip</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="communications"/>
+    <category field="social_science"/>
+    <summary>Zitierstil orientiert sich am Leitfaden zur Anfertigung von Seminar-, Diplom- und Masterarbeiten (Stand September 2015) von Prof. Weber, BWL, Universität Mannheim (der neue Leitfaden ist passwortgeschützt und nicht frei zugänglich im Netz). Die Implemantation ist sprachunabhängig, so dass der gleiche Stil für Deutsch und Englisch gebraucht werden kann.</summary>
```
Comment (stweil) [inline universitaet-mannheim-lehrstuhl-prof-weber.csl:19]: s/Implemantation/Implementation/

Comment (zuphilip) [inline universitaet-mannheim-lehrstuhl-prof-weber.csl:19]: Korrigiert in #3 . Danke!

**Pull request merged**